src/luarocks/core/cfg.lua

```diff
@@ -397,6 +397,18 @@ local function make_defaults(lua_version, target_cpu, platforms, home)
          defaults.variables.CFLAGS = defaults.variables.CFLAGS.." -fPIC"
       end
       defaults.web_browser = "xdg-open"
+      if platforms.linux then
+         -- inline code from fs/linux.lua since
+         -- luarocks.fs can't be required here
+         -- (circular dependencies)
+         local fd, _, code = io.open("/usr/lib64", "r")
+         if code ~= 2 then
+            defaults.lib_modules_path = "/lib64/lua/"..lua_version
```

Comment:
was this meant to be `/usr/lib64/lua`?

Reply:
`/usr/lib64/lua` would also work, yes. I'm just trying to distinguish between 32-bit Fedora/RHEL (/usr/lib is used) and 64-bit Fedora/RHEL (libraries goes in /usr/lib64)

Reply:
@daurnimator the `lib_modules_path` variabe is a path fragment, not an absolute path.

Reply:
> the `lib_modules_path` variable is a path fragment, not an absolute path.

Then shouldn't we `io.open(prefix .. "/lib64")` or similar?